commands.json

```diff
@@ -14807,11 +14807,11 @@
                 "key_spec_index": 0
             },
             {
-                "name": "min",
+                "name": "start",
```

Comment:
These JSON changes need to be done also in src/commands/zrange.json in the Redis main repository.